### PR TITLE
removeEmoji()

### PR DESCRIPTION
**Pull request by KarelWintersky**
* [+] added ext-mbstring dependency to composer.json
* [+] added static method `removeEmoji()`
* [+] added test method `testRemoveEmoji()`, test passed
* [*] callback declared as `static function` instead of `function`

### CODE REVIEW

**Comment by KarelWintersky** [comment]
Also, I think, we can add file `.gitattributes`:

```
* text=auto
* eol=lf

.git				export-ignore
.gitattributes      export-ignore
.gitignore          export-ignore
.github             export-ignore
tests               export-ignore
vendor              export-ignore
phpunit.xml         export-ignore
bin                 export-ignore
```
tests and generate scripts can be removed from exported package, 'cause tests and `shortcodes-array.php` required at development stage.

**Review by joshmcrae — CHANGES_REQUESTED** [review]
Thanks for this enhancement! A couple of minor changes to make before continuing but other than those I'm happy to merge and release.

The `.gitattributes` change looks acceptable too.

File: src/LitEmoji.php [hunk 1]
```diff
@@ -187,6 +187,19 @@ public static function config($property, $value)
...
         }
     }
+    
+    /**
+     * Removes all emoji-sequences from string.
+     *
+     * @param string $source
+     * @return string
+     */
+    public static function removeEmoji(string $source):string
```
[the remark this answers is not in the document]
Comment (KarelWintersky) [reply src/LitEmoji.php]: fixed

File: tests/LitEmojiTest.php [hunk 2]
```diff
@@ -47,6 +47,12 @@ public function testUnicodeToShortcodeTiming()
         $text = LitEmoji::encodeShortcode(file_get_contents(__DIR__ . '/UnicodeIpsum'));
         $this->assertEquals(file_get_contents(__DIR__ . '/ShortcodeIpsum'), $text);
     }
+    
+    public function testRemoveEmoji()
+    {
+        $text = LitEmoji::removeEmoji('Some text 😊 including emoji 🚀');
+        $this->assertEquals('Some text  including emoji ', $text); //NB: smile emoji surrounded with TWO spaces.
```
[the remark this answers is not in the document]
Comment (KarelWintersky) [reply tests/LitEmojiTest.php]: fixed

**Comment by KarelWintersky** [comment]
gitattributes added

**Review by joshmcrae — APPROVED** [review]

**Comment by joshmcrae** [comment]
Released as [4.1.0](https://github.com/elvanto/litemoji/releases/tag/4.1.0).

**Pull request merged**